smartsim/launchable/baseJobGroup.py

```diff
@@ -8,14 +10,11 @@
 from .basejob import BaseJob
 
 
-class BaseJobGroup(Launchable, MutableSequence):
```

Comment:
That was my bad, nice catch!

Reply:
How did the tests still pass even though I did not inherit the ABC class?

Reply:
Without going _too_ far into the weeds of how `ABC` and `abstractmethod` work, the `@abstractmethod` decorator just sets a `__isabstractmethod__ = True` attr to the method.

```py
class C:
    @abstractmethod
    def some_method(self):  print("Totally works!")

C().some_method()  # prints: "Totally works!"
C().some_method.__isabstractmethod__  # returns: `True`
```

The `ABC` class is what actually enforces that no methods have a `__isabstractmethod__` set to true when the class is created:
```py
class C(ABC):
    @abstractmethod
    def some_method(self): ...

C()  # Now cannot be instanced, raises a `TypeError`
```
So basically, it got past the tests precisely _because_ it did not inherit from `ABC`!